src/Domain/Creators/BinaryCompactObject.cpp

```diff
@@ -483,14 +579,16 @@ Domain<3> BinaryCompactObject::create_domain() const {
         expansion_map_outer_boundary_, expansion_function_of_time_name_,
         expansion_function_of_time_name_ + "OuterBoundary"s};
     RotationMap3D rotation_map{rotation_function_of_time_name_};
-    CompressionMap size_A_map{size_map_function_of_time_names_[0],
-                              object_A_.inner_radius,
-                              object_A_.outer_radius,
-                              {{object_A_.x_coord, 0.0, 0.0}}};
-    CompressionMap size_B_map{size_map_function_of_time_names_[1],
-                              object_B_.inner_radius,
-                              object_B_.outer_radius,
-                              {{object_B_.x_coord, 0.0, 0.0}}};
+    CompressionMap size_A_map{
+        size_map_function_of_time_names_[0],
+        use_single_block_a ? 1.0 : std::get<0>(object_A_).inner_radius,
+        use_single_block_a ? 2.0 : std::get<0>(object_A_).outer_radius,
```

Comment:
Could you add a comment why you chose values 1 and 2 here?

Reply:
We don't even use the size map unless there is an excision. So I think for clarity it'd be better to default construct the maps, and then if an object is excised, actually construct the map with the radii.